MAUI/Popup/popup-size.md

```diff
@@ -0,0 +1,333 @@
+                                <DataTemplate>
+                                    <ViewCell>
+                                        <ViewCell.View>
+                                            <Grid x:Name="grid" RowSpacing="1">
+                                                <Grid.RowDefinitions>
+                                                    <RowDefinition Height="Auto" />
+                                                </Grid.RowDefinitions>
+                                                <Grid.ColumnDefinitions>
+                                                    <ColumnDefinition Width="60" />
+                                                    <ColumnDefinition Width="200" />
+                                                    <ColumnDefinition Width="20" />
+                                                </Grid.ColumnDefinitions>
+
+                                                <Image Source="{Binding ContactImage}"
+                                       VerticalOptions="Center"
+                                       HorizontalOptions="End"
+                                       HeightRequest="40"/>
+
+                                                <Label Grid.Column="1" VerticalTextAlignment="Center"
+                                        HorizontalTextAlignment="Center"
+                                        LineBreakMode="NoWrap"
+                                        Text="{Binding ContactName}" 
+                                        FontSize="16" />
+
+                                                <Image Grid.Column="2" 
+                                       Source="{Binding ContactType}"
+                                       VerticalOptions="Center"
+                                       HorizontalOptions="End"
+                                       HeightRequest="15"/>
+
+                                            </Grid>
+                                        </ViewCell.View>
+                                    </ViewCell>
+                                </DataTemplate>
+                            </listView:SfListView.ItemTemplate>
+                        </listView:SfListView>
+                    </DataTemplate>
+                </popup:SfPopup.ContentTemplate>
+            </popup:SfPopup>
+        </StackLayout>
+    </ContentPage.Content>
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+using Syncfusion.Maui.ListView;
+using Syncfusion.Maui.Popup;
+
+namespace PopupMaui
+{
+    public partial class MainPage : ContentPage
+    {
+         SfListView listView;
+        ContactsViewModel viewModel;
+        SfPopup popup;
+
+        public MainPage()
+        {
+            InitializeComponent();
+            listView = new SfListView() { ItemSpacing = 5 };
+            listView.WidthRequest = 350;
+            listView.ScrollBarVisibility = ScrollBarVisibility.Never;
+            listView.ItemTemplate = new DataTemplate(() =>

```

Comment:
No Need to show full codes.
Highlight required lines.

Reply:
Addressed the concerns provided.